.infrastructure/pvc.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: todoapp-pvc
+  namespace: todoapp
+spec:
+  volumeName: todoapp-pv
```

Comment:
Specifying `volumeName` directly binds this PVC to a specific PV, which can limit flexibility and dynamic provisioning capabilities. Consider removing `volumeName` to allow Kubernetes to automatically bind the PVC to an appropriate PV based on the storage class and other criteria.